pad_server/src/state.h

```diff
@@ -11,11 +11,21 @@
 /* Number of actuators in the system: 12 solenoid valves, 1 fire valve, 1 quick disconnect, 1 igniter */
 #define NUM_ACTUATORS (12 + 1 + 1 + 1)
 
+typedef struct {
+    pthread_mutex_t mut;
+    pthread_cond_t cond;
+    enum { ACT, ARM } target;
```

Comment:
This isn't necessary; whenever anything is updated (actuators or arming level), we re-send ALL the information (both actuator states and arming level). If we timeout while waiting for an update, we also re-send all the data again.